### PR TITLE
Use absolute imports

### PR DESCRIPTION
**Pull request by suzil**
* Absolute imports are easier to read and allow files to be moved around without breaking them

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,8 @@
     install_requires=[
         'six>=1.8.0',
         'w3lib>=1.15.0',
-        'cityhash>=0.1.7'
+        'cityhash>=0.1.7',
+        "thriftpy"
```
Comment (sibiryakov) [inline setup.py]: thriftpy is only needed when installed with hbase and distributed stuff

Comment (suzil) [reply setup.py]: Tests broke when I didn't include it, should it be in a different section for installs?

Comment (sibiryakov) [reply setup.py]: and yes, thtiftpy2

**Comment by sibiryakov** [comment]
Conflict @suzil

File: setup.py (hunk 1 above)
Comment (suzil) [reply setup.py]: Fixed

**Comment by suzil** [comment]
Fixed now @sibiryakov

**Comment by suzil** [comment]
closing due to no response

**Pull request closed without merging**